controllers/controlplane_controller_consts.go

```diff
@@ -0,0 +1,15 @@
+package controllers
+
+// -----------------------------------------------------------------------------
+// ControlPlane - Finalizers
+// -----------------------------------------------------------------------------
+
+// ControlPlaneFinalizer are possible finalizers added by controlplane controller.
```

Comment:
nit: (although I'm not sure this fits 100%)

```suggestion
// ControlPlaneFinalizer defines finalizers added by controlplane controller.
```

Reply:
updated.